src/tests/java/htsjdk/variant/vcf/VCFHeaderUnitTest.java

```diff
@@ -131,6 +134,64 @@ public void testVCFHeaderSampleRenamingSitesOnlyVCF() throws Exception {
         final VCFHeader header = (VCFHeader) codec.readHeader(vcfIterator).getHeaderValue();
     }
 
+    @Test
+    public void testVCFHeaderAddMetaDataLine() {
```

Comment:
This should really be called something like `testVCFHeaderAddMetaDataLineDoesNotDuplicateContigs()`, since it's not actually testing whether the new line was added correctly.